confignode/src/main/java/org/apache/iotdb/confignode/service/ConfigNode.java

```diff
@@ -305,7 +311,8 @@ private void sendRestartConfigNodeRequest() throws IOException, StartupException
     TEndPoint targetConfigNode = CONF.getTargetConfigNode();
     if (targetConfigNode == null) {
       LOGGER.error(
-          "Please set the cn_target_config_node_list parameter in iotdb-confignode.properties file.");
+          "Please set the cn_target_config_node_list parameter in "
```

Comment:
Maybe we don't need to change the code smell of long lines of code because it will lead to more string concatenation